doc/source/supported-cloud-providers.csv

```diff
@@ -6,3 +6,4 @@ OVH,OVH,``ovh``
 OpenTelekomCloud,T-Systems,``otc``
 Wavestack,noris network,``wavestack``
 pluscloud open,plusserver,``pluscloudopen``
+f1a,cloudandheat,``cah-f1a``
```

Comment:
Use ``Cloud&Heat `` instead of ``cloudandheat`` and use ``cloudandheat-f1a`` instead of ``cah-f1a``.

Is ``f1a`` really the name of this cloud?

Reply:
The name should probably be `Cloud&Heat Datacenter Frankfurt` or similar, but I'm not sure we're a good fit for supported-cloud-providers.csv because you need to go through sales to even get an IaaS account with us.

(I still see value to have us listed in `clouds-public.yaml` and to have the terraform template there in order to make it easier for us to contribute to SCS.)

Reply:
The clouds-public.yaml is only a hack because the gopherclient library isn't able to read it from a standard location. You can easily use the testbed without it by simply adding all the relevant items into your clouds.yaml directly.